### ISSUE TITLE
Don't kill players inside indestructible blocks if they have fly or noclip privs

### ISSUE THREAD

**Issue by farooqkz** [MT-CTF/capturetheflag#1280]
It is useful when testing maps and for admin as well. It's annoying for me when I try to navigate in a map and get killed instantly.

**Comment by HobbitPower** [comment]
Simply uncheck the "enable damage" box

**Comment by farooqkz** [comment]
> Simply uncheck the "enable damage" box. At least for map testing it fulfils its purpose.

Good idea. But what about when the server admin is online on the server?

**Comment by LoneWolfHT** [comment]
I'll implement this since it's annoying when spectating during tournaments
